src/lib/viewers/error/PreviewErrorViewer.js

```diff
@@ -80,8 +80,7 @@ class PreviewErrorViewer extends BaseViewer {
 
         const { displayMessage, details, message } = error;
         const { file } = this.options;
-        const { extension } = file || {};
-        const iconExtension = extension === 'flv' || extension === 'tgz' || extension === 'zip' ? extension : undefined;
+        const { extension: iconExtension } = file || {};
```

Comment:
what happens if `iconExtension` is undefined?

Reply:
These scenarios will be handled by [getIcon](https://github.com/box/box-ui-elements/blob/1850977829e4888139462a6f65794b1cffc52a08/src/components/preview/previewIcons.ts#L183) in box-ui-elements. In the case of undefined, it will return the same default item as it returned before when iconExtension was explicitly set to undefined.